src/routes/app/(app)/components/load-default-explore-page-data.ts

```diff
@@ -0,0 +1,132 @@
+import { PUBLIC_NETWORK } from '$env/static/public';
+import { gql } from 'graphql-request';
+import {
+  DEFAULT_EXPLORE_PAGE_FEATURED_DRIP_LISTS_FRAGMENT,
+  DEFAULT_EXPLORE_PAGE_FEATURED_PROJECT_FRAGMENT,
+} from './default-explore-page.svelte';
+import { postsListingSchema } from '../../../api/blog/posts/schema';
+import mapFilterUndefined from '$lib/utils/map-filter-undefined';
+import { cachedTotalDrippedPrices } from '$lib/utils/total-dripped-approx';
+import { redis } from '../../../api/redis';
+import cached from '$lib/utils/cache/remote/cached';
+import query from '$lib/graphql/dripsQL';
+import queryCacheKey from '$lib/utils/cache/remote/query-cache-key';
+import type {
+  ExploreProjectsQuery,
+  ExploreProjectsQueryVariables,
+  FeaturedDripListQuery,
+  FeaturedDripListQueryVariables,
+} from './__generated__/gql.generated';
+import {
+  ProjectSortField,
+  ProjectVerificationStatus,
+  SortDirection,
+} from '$lib/graphql/__generated__/base-types';
+import network from '$lib/stores/wallet/network';
+
+const FEATURED_DRIP_LISTS =
+  {
+    1: [
```

Comment:
What do 1 and 5 mean is this context? I notice you used them in an earlier file too.

Reply:
those are chain IDs — I will add explicit typing to that var so it's clear